Binary files gdm-3.38.1.orig/.meson.build.un~ and gdm-3.38.1/.meson.build.un~ differ
diff -uNr gdm-3.38.1.orig/meson.build gdm-3.38.1/meson.build
--- gdm-3.38.1.orig/meson.build	2020-10-13 16:13:18.834374069 -0300
+++ gdm-3.38.1/meson.build	2020-10-13 16:49:27.806607739 -0300
@@ -94,9 +94,9 @@
   libwrap_dep = cc.find_library('libwrap')
 endif
 # systemd
-systemd_dep = dependency('systemd')
-libsystemd_dep = dependency('libsystemd')
-if meson.version().version_compare('>= 0.53')
+systemd_dep = dependency('libelogind')
+libsystemd_dep = dependency('libelogind')
+if meson.version().version_compare('>= 0.53')
   systemd_multiseat_x = find_program('systemd-multi-seat-x',
     required: false,
     dirs: [
